### PR TITLE
align relay log behaviour in v5.4.0

### PR DESCRIPTION
**Pull request by lance6716**
### What is changed, added or deleted? (Required)

as title

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: dm/relay-log.md [hunk 1]
```diff
@@ -95,17 +95,41 @@ Relay log 迁移的起始位置由如下规则决定：
...
+```
+
+</div> 
+    
+<div label="v2.0.2（包含）到 v5.3.0（包含）">
 
 > **注意：**
 > 
-> 自 v2.0.2 起，上游数据源配置中的 `enable-relay` 项已经失效。在[加载数据源配置](/dm/dm-manage-source.md#数据源操作)时，如果发现配置中的 `enable-relay` 项为 `true`，DM 会给出如下信息提示：
+> 在 v2.0.2（包含）到 v5.3.0（包含）中，上游数据源配置中的 `enable-relay` 项已经失效。[加载数据源配置](/dm/dm-manage-source.md#数据源操作)时，如果发现配置中的 `enable-relay` 项为 `true`，DM 会给出如下信息提示：
```
Comment (sunzhaoyang) [inline dm/relay-log.md:126]:
```suggestion
> 在 v2.0.2（包含）至 v5.3.0（包含）版本，上游数据源配置中的 `enable-relay` 项失效，仅能通过`start-relay`/`stop-relay`命令开启和关闭。[加载数据源配置](/dm/dm-manage-source.md#数据源操作)时，如果发现配置中的 `enable-relay` 项为 `true`，DM 会给出如下信息提示：
```

File: dm/relay-log.md [hunk 2]
```diff
@@ -95,17 +95,41 @@ Relay log 迁移的起始位置由如下规则决定：
...
 > 
 > ```
 > Please use `start-relay` to specify which workers should pull relay log of relay-enabled sources.
 > ```
 
-在 v2.0.2 及之后的版本中，`start-relay` 与 `stop-relay` 命令分别用于启动及停止 relay log 的拉取。
+在 v2.0.2（包含）到 v5.3.0（包含）中，`start-relay` 与 `stop-relay` 命令分别用于启动及停止 relay log 的拉取。
```
Comment (sunzhaoyang) [inline dm/relay-log.md:133]:
```suggestion
```

**Comment by sunzhaoyang** [comment]
rest  LGTM

File: dm/relay-log.md [hunk 3]
```diff
@@ -8,7 +8,7 @@ aliases: ['/docs-cn/tidb-data-migration/dev/relay-log/']
 
 DM (Data Migration) 工具的 relay log 由若干组有编号的文件和一个索引文件组成。这些有编号的文件包含了描述数据库更改的事件。索引文件包含所有使用过的 relay log 的文件名。
 
-在启用 relay log 功能后，DM-worker 会自动将上游 binlog 迁移到本地配置目录（若使用 TiUP 部署 DM，则迁移目录默认为 `<deploy_dir> / <relay_log>`）。本地配置目录 `<relay_log>` 的默认值是 `relay-dir`，可在[上游数据库配置文件](/dm/dm-source-configuration-file.md)中进行修改）。DM-worker 在运行过程中，会将上游 binlog 实时迁移到本地文件。DM-worker 的 sync 处理单元会实时读取本地 relay log 的 binlog 事件，将这些事件转换为 SQL 语句，再将 SQL 语句迁移到下游数据库。
+在启用 relay log 功能后，DM-worker 会自动将上游 binlog 迁移到本地配置目录（若使用 TiUP 部署 DM，则迁移目录默认为 `<deploy_dir> / <relay_log>`）。本地配置目录 `<relay_log>` 的默认值是 `relay-dir`，可在[上游数据库配置文件](/dm/dm-source-configuration-file.md)中进行修改。自 v5.4.0 版本起，本地配置目录也可以在 DM-worker 配置文件中通过 `relay-dir` 进行配置，而且其相比于上游数据库配置文件具有更高优先级）。DM-worker 在运行过程中，会将上游 binlog 实时迁移到本地文件。DM-worker 的 sync 处理单元会实时读取本地 relay log 的 binlog 事件，将这些事件转换为 SQL 语句，再将 SQL 语句迁移到下游数据库。
```
Comment (TomShawn) [inline dm/relay-log.md:11]:
```suggestion
在启用 relay log 功能后，DM-worker 会自动将上游 binlog 迁移到本地配置目录（若使用 TiUP 部署 DM，则迁移目录默认为 `<deploy_dir> / <relay_log>`）。本地配置目录 `<relay_log>` 的默认值是 `relay-dir`，可在[上游数据库配置文件](/dm/dm-source-configuration-file.md)中进行修改。自 v5.4.0 版本起，你可以在 DM-worker 配置文件中通过 `relay-dir` 配置本地配置目录，其优先级高于上游数据库的配置文件。

DM-worker 在运行过程中，会将上游 binlog 实时迁移到本地文件。DM-worker 的 sync 处理单元会实时读取本地 relay log 的 binlog 事件，将这些事件转换为 SQL 语句，再将 SQL 语句迁移到下游数据库。
```

File: dm/relay-log.md [hunk 4]
```diff
@@ -95,17 +95,41 @@ Relay log 迁移的起始位置由如下规则决定：
...
 <SimpleTab>
-<div label="v2.0.2 及之后的版本">
+
+<div label="v5.4.0 及之后的版本">
+
+在 v5.4.0 及之后的版本中，DM-worker 在绑定上游数据源时，会检查上游数据源配置中的 `enable-relay` 项。如果 `enable-relay` 为 `true`，则为该数据源启用 relay log 功能。
+
+具体配置方式参见[上游数据源配置文件介绍](/dm/dm-source-configuration-file.md)
+    
+除此以外，也可以通过 `start-relay`/`stop-relay` 命令动态调整数据源的 `enable-relay` 并即时开启或关闭 relay log。
```
Comment (TomShawn) [inline dm/relay-log.md:105]:
```suggestion
除此以外，也可以通过 `start-relay` 或 `stop-relay` 命令动态调整数据源的 `enable-relay` 并即时开启或关闭 relay log。
```

File: dm/relay-log.md [hunk 5]
```diff
@@ -95,17 +95,41 @@ Relay log 迁移的起始位置由如下规则决定：
...
+    
+{{< copyable "" >}}
```
Comment (TomShawn) [inline dm/relay-log.md:107]:
```suggestion
{{< copyable "shell-regular" >}}
```

File: dm/relay-log.md [hunk 6]
```diff
@@ -95,17 +95,41 @@ Relay log 迁移的起始位置由如下规则决定：
...
+
+```bash
+» start-relay -s mysql-replica-01
```
Comment (TomShawn) [inline dm/relay-log.md:110]:
```suggestion
start-relay -s mysql-replica-01
```

File: dm/relay-log.md [hunk 7]
```diff
@@ -95,17 +95,41 @@ Relay log 迁移的起始位置由如下规则决定：
...
+```
+
+</div> 
+    
+<div label="v2.0.2（包含）到 v5.3.0（包含）">
 
 > **注意：**
 > 
-> 自 v2.0.2 起，上游数据源配置中的 `enable-relay` 项已经失效。在[加载数据源配置](/dm/dm-manage-source.md#数据源操作)时，如果发现配置中的 `enable-relay` 项为 `true`，DM 会给出如下信息提示：
+> 在 v2.0.2（包含）至 v5.3.0（包含）版本，上游数据源配置中的 `enable-relay` 项失效，仅能通过`start-relay`/`stop-relay`命令开启和关闭。[加载数据源配置](/dm/dm-manage-source.md#数据源操作)时，如果发现配置中的 `enable-relay` 项为 `true`，DM 会给出如下信息提示：
```
Comment (TomShawn) [inline dm/relay-log.md:126]:
```suggestion
> 在 v2.0.2 及之后的 v2.0 版本，以及 v5.3.0 版本中，上游数据源配置中的 `enable-relay` 项失效，你只能通过`start-relay` 和 `stop-relay`命令开启和关闭 relay log。[加载数据源配置](/dm/dm-manage-source.md#数据源操作)时，如果你发现配置中的 `enable-relay` 项为 `true`，DM 会给出如下信息提示：
```

File: dm/relay-log.md [hunk 8]
```diff
@@ -95,17 +95,41 @@ Relay log 迁移的起始位置由如下规则决定：
 ## 启动、停止 relay log
 
 <SimpleTab>
-<div label="v2.0.2 及之后的版本">
+
+<div label="v5.4.0 及之后的版本">
+
+在 v5.4.0 及之后的版本中，DM-worker 在绑定上游数据源时，会检查上游数据源配置中的 `enable-relay` 项。如果 `enable-relay` 为 `true`，则为该数据源启用 relay log 功能。
```
Comment (TomShawn) [inline dm/relay-log.md:101]:
```suggestion
在 v5.4.0 及之后的版本中，你可以通过将 `enable-relay` 设为 `true` 开启 relay log。自 v5.4.0 起，DM-worker 在绑定上游数据源时，会检查上游数据源配置中的 `enable-relay` 项。如果 `enable-relay` 为 `true`，则为该数据源启用 relay log 功能。
```

File: dm/relay-log.md (hunk 6 above)
Comment (lance6716) [reply dm/relay-log.md:112]: 这个其实不是 shell 命令，而是进入 dmctl 之后的交互

File: dm/relay-log.md (hunk 7 above)
Comment (lance6716) [reply dm/relay-log.md:128]: 这里不是 “如果你发现配置中的 `enable-relay` 项为 `true`”，想表达 “如果 DM 发现配置中的 `enable-relay` 项为 `true`”

File: dm/relay-log.md (hunk 2 above)
Comment (lance6716) [reply dm/relay-log.md:135]: 要删掉吗，为啥（这个是唯一启动 relay log 的方式）

File: dm/relay-log.md (hunk 7 above)
Comment (TomShawn) [reply dm/relay-log.md:128]: 谢谢，已修改 comment

File: dm/relay-log.md (hunk 2 above)
Comment (sunzhaoyang) [reply dm/relay-log.md:135]: 我是觉得跟前面128行有点重复了，留一份就可以了吧？

**Review by TomShawn — APPROVED** [review]
LGTM

**Review by sunzhaoyang — APPROVED** [review]
LGTM

**Pull request merged**